android/src/main/kotlin/com/wire/kalium/KaliumApplication.kt

```diff
@@ -39,7 +39,8 @@ class KaliumApplication : Application(), Configuration.Provider {
         coreLogic = CoreLogic(
             appContext = applicationContext,
             rootPath = rootDir.absolutePath,
-            kaliumConfigs = KaliumConfigs()
+            kaliumConfigs = KaliumConfigs(),
+            userAgent = "Kalium test app"
```

Comment:
Could we have the same as in darwin for example?
like: `userAgent = "Kalium Android"`